.github/workflows/spell-checker.yml

```diff
@@ -0,0 +1,42 @@
+name: Spell Checker
+on:
+  pull_request_target:
+    branches:
+    - "**"
+    types:
+    - 'opened'
+    - 'reopened'
+    - 'synchronize'
+
+jobs:
+  spelling:
+    name: Check Spelling
+    permissions:
+      contents: read
+      pull-requests: read
+      actions: read
+    outputs:
+      followup: ${{ steps.spelling.outputs.followup }}
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event_name, 'pull_request')}}
+    concurrency:
+      group: spelling-${{ github.event.pull_request.number || github.ref }}
+      # note: If you use only_check_changed_files, you do not want cancel-in-progress
+      cancel-in-progress: true
+    steps:
+    - name: check-spelling
+      id: spelling
+      uses: check-spelling/check-spelling@main
+      with:
+        suppress_push_for_open_pull_request: ${{ github.actor != 'dependabot[bot]' && 1 }}
+        checkout: true
+        check_file_names: 1
+        spell_check_this: check-spelling/spell-check-this@prerelease
```

Comment:
please use the latest released version here and not `prerelease`

Reply:
@mowies I can't find the version of `check-spelling/spell-check-this`.
May be it is currently in pre-release only

Reply:
i double checked, there's no tags, just branches. please use `main` branch instead of `prerelease` :)

Reply:
```suggestion
        spell_check_this: check-spelling/spell-check-this@main
```